src/components/main/StaffList.js

```diff
@@ -4,12 +4,56 @@ import './StaffList.css';
 
 class StaffList extends React.Component {
    
-  
+    listAdd = (sf,index) =>{
+       let quantity = parseInt(this.props.staffs[sf].quantity)
+       let shiftTotal = parseInt(this.props.staffs[sf].shiftTotal)
+       let shift = parseInt(this.props.staffs[sf].shift)
+        quantity = quantity + 1
+
+        if (shift == 12) {
+            shiftTotal = parseInt(shiftTotal) + 12;
+        }else{
+            shiftTotal = parseInt(shiftTotal) + 8;
```

Comment:
I think I understand what +8 or -8 is doing. Since we are also doing a sub calculation it needs to update the subtotal for the record. Though I'm not sure if this is how we should handle it? Maybe you can explain in one of our meetings? I will do some thinking/research as well my one thought about doing it this way is that later if we attach more meaning to a shift type. Then this might not give the desired change that we are expecting.